docs/CMakeLists.txt

```diff
@@ -79,4 +79,8 @@ if (LATEX_PDFLATEX)
   if (LATEX_BUILD)
     add_subdirectory(latex)
   endif()
+else()
+  if (LATEX_BUILD)
```

Comment:
might need to flip the logic of these if blocks and move `option(LATEX_BUILD ...` up so that it is defined in this scope

Reply:
So, should we do that? If so, I'm happy to do so, but I don't understand exactly what you mean. Here's the code:

```
# changing this variable does not do anything
find_program(LATEX_PDFLATEX pdflatex)
if (LATEX_PDFLATEX)
  # build pdfs from latex files (default off to not require LaTeX)
  option(LATEX_BUILD "Enable building of LaTeX documentation" Off)
  message(STATUS "pdflatex found at ${LATEX_PDFLATEX}. Adding LaTeX building option (currently ${LATEX_BUILD}).")
  if (LATEX_BUILD)
    add_subdirectory(latex)
  endif()
else()
  if (LATEX_BUILD)
    message(FATAL_ERROR "pdflatex executable not found. Please install in order to build GUFI documentation.")
  endif()
endif()
```

How do you want it changed?

Reply:
I think you need to have `option(LATEX_BUILD ...` at the top, and if it is true, look for pdflatex with `find_program ...` so that if pdflatex isnt found, the error is triggered

Reply:
OK. I think I did what you suggested. Please LMK. Thanks.